.eslintrc.js

```diff
@@ -1,12 +1,15 @@
 module.exports = {
   root: true,
-  extends: "@react-native-community",
+  extends: ["@react-native-community", "plugin:i18next/recommended"],
   rules: {
     quotes: [2, "double"],
     "comma-dangle": [2, "never"],
     "space-in-parens": [2, "always"],
     "prettier/prettier": 0,
-    "no-var": 1
+    "no-var": 1,
+    // TODO: remove this line once we have image assets and no longer need
+    // placeholder text; better to enforce globalized text with errors, not warnings
+    "i18next/no-literal-string": 1
```

Comment:
I'm concerned that we'll do this and never actually start enforcing this, so instead of making this a warning, why don't we make it an error and either remove all placeholder text replace all it all with something like `<PlaceholderText text="this is just a placeholder" />` and that component is just something like 

```javascript
const PlaceholderText = ( { text } ) => (
  // eslint-disable-next-line react-native/no-inline-styles
  <Text style={{ color: "red", textTransform: "uppercase", fontSize: 30 }}>
    { text }
  </Text>
);
```

That way if you *really* want to add placeholder text you can, but you have be more self-conscious about it, and we can find and remove it a little more easily. Very tempted to install [`react-blink-tag`](https://www.npmjs.com/package/react-blink-tag), but that's probably overkill, if amusing.